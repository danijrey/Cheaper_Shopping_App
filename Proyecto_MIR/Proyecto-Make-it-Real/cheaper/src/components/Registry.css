.App {
	width: 100vmin;
	height: 100vmin;
	text-align: center;
	background-color: white;
}

.PaginadeRegistro {
	width: 100vw;
	height: 100vh;
	background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.1)),
		url("../assets/Carros.jpg") no-repeat center top;
	background-size: cover;
	color: white;
	display: flex;
	flex-direction: column;
	align-items: center;
}

.NavBar-Logo-Registry {
	width: 300px;
	margin: 20px;
}

.PaginaDeRegistro-Formurlario {
	width: 270px;
	height: 290px;
	margin: 20px;
	padding: 10px;
	background-color: rgba(48, 48, 48, 0.8);
	border-radius: 10px;
}

.Bienvenido {
	font-size: 24px;
	margin-bottom: 5px;
}
.Frase {
	font-weight: 600;
	margin: 5px;
}

.Botones-Ingreso {
	margin-top: 18px;
	display: flex;
	justify-content: center;
	display: grid;
	grid-template-columns: 250px;
	grid-template-rows: 40px 40px 40px;
	grid-gap: 18px;
}

.Boton-IniciarSesion {
	border-radius: 20px;
	padding-top: 7px;
	font-weight: 600;
	font-size: 15px;
	background: #e0c408c9;
	border-color: black;
	color: white;
	cursor: pointer;
}

.Boton-IniciarSesion:hover {
	background: #e0c308;
}

.Boton-Registrarse {
	border-radius: 20px;
	padding-top: 7px;
	font-weight: 600;
	font-size: 15px;
	background: #e0c408c9;
	border-color: black;
	color: white;
	cursor: pointer;
	text-decoration: none;
}

.Boton-Registrarse:hover {
	background: #e0c308;
}

.Boton-CrearEmpresa {
	border-radius: 20px;
	padding-top: 7px;
	font-weight: 600;
	font-size: 15px;
	background: #e0c408c9;
	border-color: black;
	color: white;
	cursor: pointer;
	text-decoration: none;
}

.Boton-CrearEmpresa:hover {
	background: #e0c308;
}

@media screen and (min-width: 768px) {
	.PaginadeRegistro {
		width: 768px;
		height: 529px;
	}
	.PaginaDeRegistro-Formurlario {
		width: 290px;
		height: 300px;
	}
}

@media screen and (min-width: 992px) {
	.PaginadeRegistro {
		width: 1024px;
		height: 529px;
	}

	/* .NavBar-Logo {
		width: 300px;
		margin: 15px;
	} */

	.PaginaDeRegistro-Formurlario {
		width: 350px;
		height: 350px;
	}
	.Botones-Ingreso {
		grid-gap: 30px;
	}

	.Bienvenido {
		font-size: 30px;
	}

	.Frase {
		font-weight: 500;
	}
}

@media screen and (min-width: 1200px) {
	.PaginadeRegistro {
		width: 1440px;
		height: 620px;
	}

	.NavBar-Logo-Registry {
		width: 400px;
		margin: 15px;
	}

	.PaginaDeRegistro-Formurlario {
		width: 350px;
		height: 300px;
	}
	.Botones-Ingreso {
		grid-gap: 30px;
	}

	.Bienvenido {
		font-size: 30px;
	}

	.Frase {
		font-weight: 500;
	}
}
