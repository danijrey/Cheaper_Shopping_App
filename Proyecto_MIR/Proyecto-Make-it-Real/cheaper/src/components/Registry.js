import React from "react";
import logo from "../assets/logo.png";
import "./Registry.css";
import { Link } from "react-router-dom";

class Registry extends React.Component {
	render() {
		return (
			<div className="App">
				<div className="PaginadeRegistro">
					<img src={logo} className="NavBar-Logo-Registry" alt="Logo"></img>
					<div className="PaginaDeRegistro-Formurlario">
						<h1 className="Bienvenido"> Bienvenido a Cheaper </h1>
						<p className="Frase">Encuentra las opciones más económicas </p>
						<div className="Botones-Ingreso">
							<Link to="/signin" className="Boton-IniciarSesion">
								Iniciar Sesión
							</Link>
							<Link to="/clients/create" className="Boton-Registrarse">
								Registrarse
							</Link>
							<Link to="/providers/create" className="Boton-CrearEmpresa">
								Crear una cuenta de empresa
							</Link>
						</div>
					</div>
				</div>
			</div>
		);
	}
}

export default Registry;
