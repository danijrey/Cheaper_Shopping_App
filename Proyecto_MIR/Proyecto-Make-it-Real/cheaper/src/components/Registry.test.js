import React from 'react';
import {render} from '@testing-library/react';
import {MemoryRouter} from 'react-router-dom';
import Registry from './Registry';


describe ('Registry',() =>{
    it ('should render correctly', ()=>{
    const {container} = render (
    <MemoryRouter>
        <Registry/>
    </MemoryRouter> 
    
    ); 
    expect (container.firstChild).toMatchSnapshot();
    });
});