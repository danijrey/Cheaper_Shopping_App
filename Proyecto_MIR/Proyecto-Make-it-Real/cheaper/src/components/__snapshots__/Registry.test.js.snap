// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registry should render correctly 1`] = `
<div
  class="App"
>
  <div
    class="PaginadeRegistro"
  >
    <img
      alt="Logo"
      class="NavBar-Logo-Registry"
      src="logo.png"
    />
    <div
      class="PaginaDeRegistro-Formurlario"
    >
      <h1
        class="Bienvenido"
      >
         Bienvenido a Cheaper 
      </h1>
      <p
        class="Frase"
      >
        Encuentra las opciones más económicas 
      </p>
      <div
        class="Botones-Ingreso"
      >
        <a
          class="Boton-IniciarSesion"
          href="/signin"
        >
          Iniciar Sesión
        </a>
        <a
          class="Boton-Registrarse"
          href="/clients/create"
        >
          Registrarse
        </a>
        <a
          class="Boton-CrearEmpresa"
          href="/providers/create"
        >
          Crear una cuenta de empresa
        </a>
      </div>
    </div>
  </div>
</div>
`;
